src/main/java/by/antonsh/lesson10/Ex29.java

```diff
@@ -0,0 +1,21 @@
+package by.antonsh.lesson10;
+
+import java.util.ArrayList;
+import java.util.ListIterator;
+import java.util.Random;
+import java.util.stream.Collectors;
+
+/*
+Создать список оценок учеников с помощью ArrayList,
+заполнить рандом. Удалить неудовлетворительные оценки
+ */
+public class Ex29 {
+    public static void main(String[] args) {
+        Random random=new Random();
+        ArrayList<Integer> grade=new ArrayList<Integer>(
```

Comment:
work at the interface level: List<Integer> grade = new ArrayList

Reply:
Ok,thks